doc/classes/Light3D.xml

```diff
@@ -55,6 +55,7 @@
 		</member>
 		<member name="light_angular_distance" type="float" setter="set_param" getter="get_param" default="0.0">
 			The light's angular size in degrees. Increasing this will make shadows softer at greater distances. Only available for [DirectionalLight3D]s. For reference, the Sun from the Earth is approximately [code]0.5[/code].
+			[b]Note:[/b] [member light_angular_distance] is not affected by [member Node3D.scale] (on the light itself or its parents).
```

Comment:
```suggestion
			[b]Note:[/b] [member light_angular_distance] is not affected by [member Node3D.scale] (The light's scale or its parent's scale).
```

Reply:
I also think skyace's suggestion is slightly better (except "The" should have a lowercase "t"